ui/app/config/config.ts

```diff
@@ -0,0 +1,7 @@
+export var Configuration = {
+  authentication: {
+    PEERDB_PASSWORD: process.env.PEERDB_PASSWORD,
+    // Set this in production to a static value
+    NEXTAUTH_SECRET: process.env.NEXTAUTH_SECRET,
```

Comment:
does this need to be set in `docker-compose.yaml` and `docker-compose-dev.yaml`?

Reply:
Yes

Reply:
can we add static values to those files as apart of this PR? Something like `changeme` or something?

Reply:
added